.azure-pipelines/templates/documentation_deploy.yml

```diff
@@ -0,0 +1,49 @@
+jobs:
+  - job: 'documentation'
+    displayName: 'Documentation'
+    pool:
+      vmImage: 'ubuntu-latest'
```

Comment:
Any particular reason why you have `latest` here and `18.04` in `documentation_build`?

Reply:
Documentation build has to run Scipp, deployment does not. It is better to stick to a single platform to avoid issues.